spec/rails_helper.rb

```diff
@@ -60,3 +51,12 @@
   # config.filter_gems_from_backtrace("gem name")
   config.include FactoryBot::Syntax::Methods
 end
+
+require 'shoulda/matchers'
```

Comment:
acturelly `validate_presence_of` need `shouda/mathers`. but require is need to write top of file.